CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
+    [pull/910]: https://github.com/cloudflare/wrangler/pull/910
+    [issue/889]: https://github.com/cloudflare/wrangler/issues/889
+
+  - **Improve error message for `wrangler init --site` when wrangler.toml already exists - [ashleygwilliams], [issue/648] [pull/931]**
+
+    `wrangler init` generally expects that you don't already have a wrangler.toml present; however it is common that users want to add static site functionality to their existing wrangler project and will try using `wrangler init` to do so. Rather than simply complaining that the toml already exists, this PR adds the workers-site directory to the project, and prints out the suggested configuration to add to `wrangler.toml`. Much nicer!
+
+    [ashleygwilliams]: https://github.com/ashleygwilliams
+    [pull/931]: https://github.com/cloudflare/wrangler/pull/931
+    [issue/648]: https://github.com/cloudflare/wrangler/issues/648
+
+  - **Add a helpful error message on authentication error - [EverlastingBugstopper], [issue/492] [pull/932]**
+
+    Previously if wrangler got back an authentication error on publish, it would just print it to the screen. This PR adds a helpful hint to users to re-run `wrangler config` to fix the error.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/932]: https://github.com/cloudflare/wrangler/pull/932
+    [issue/492]: https://github.com/cloudflare/wrangler/issues/492
+
+  - **Provide helpful error when user accidentally puts kv-namespace config under [site] - [gabbifish], [issue/798] [pull/937]**
+
+    TOML formatting can be tricky, specifically tables, so it is common for users unfamiliar with the format to accidentally nest attributes inside tables without intending it. In this case, if a user adds a kv-namespaces entry to the bottom of a toml with [site] configuration already declared, it is parsed as a part of the [site] table, rather than as a top-level key. The error output from this is not super helpful, as it just says "unknown field `kv-namespaces`" which isn't precisely correct.
+
+    This PR detects when this error occurs and provides a help suggestion to put kv-namespaces ABOVE the [site] table entry to fix the problem.
+
+    [gabbifish]: https://github.com/gabbifish
+    [pull/937]: https://github.com/cloudflare/wrangler/pull/937
+    [issue/798]: https://github.com/cloudflare/wrangler/issues/798
+
+- ### Maintenance
+
+  - **Make Azure use latest `rustc` - [EverlastingBugstopper], [issue/887] [pull/893]**
+
+    Updates our CI to update the rust toolchain to the latest stable version after installation.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/893]: https://github.com/cloudflare/wrangler/pull/893
+    [issue/887]: https://github.com/cloudflare/wrangler/issues/887
+
+  - **Fix nightly builds - [EverlastingBugstopper], [pull/895], [pull/898]**
+
+    We caught a bug in our CI that was causing nightly builds to fail in Azure Pipelines
```

Comment:
I'm actually fairly certain that we were never actually _using_ nightly builds in our nightly tests